autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -414,7 +422,7 @@ def infer_objects(self, X: pd.DataFrame) -> pd.DataFrame:
                     # In the case train data was interpreted as int
                     # and test data was interpreted as float, because of 0.0
                     # for example, honor training data
-                    X[key] = X[key].applymap(np.int64)
+                    X[key] = X[key].astype(np.int64)
```

Comment:
should we remove the if-else statement? as we are treating them equally

Reply:
Yep, nice catch.